pkg/kademlia/kademlia.go

```diff
@@ -299,20 +321,12 @@ func GetIntroNode(addr string) (*pb.Node, error) {
 	}, nil
 }
 
-// StartRefresh occasionally refreshes stale kad buckets
-func (k *Kademlia) StartRefresh(ctx context.Context) {
-	go func() {
-		err := k.RunRefresh(ctx)
-		if err != nil && err != context.Canceled {
-			k.log.Error("refresh returned", zap.Error(err))
-		}
-	}()
-}
-
 // RunRefresh occasionally refreshes stale kad buckets
 func (k *Kademlia) RunRefresh(ctx context.Context) error {
+	k.lookups.Add(1)
+	defer k.lookups.Done()
+
 	ticker := time.NewTicker(5 * time.Minute)
-	time.Sleep(time.Duration(rand.Intn(300)) * time.Second) //stagger
```

Comment:
Do you know what the point of this was originally ?

Reply:
I think it might have been to avoid starting refresh during bootstrap.